core/controllers/feedback.py

```diff
@@ -24,11 +24,13 @@
 
 transaction_services = models.Registry.import_transaction_services()
 
+
 class ThreadListHandler(base.BaseHandler):
     """Handles operations relating to feedback thread lists."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.open_access
```

Comment:
Editor page is visible to even guest user for a public exploration. So, feedbacks should also be visible to them but they should not be able to edit the threads. 
(should I rename `can_view_exp_feedback` to something else (maybe `can_comment_on_feedback_thread`) ?)